addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -2,39 +2,114 @@ odoo.define('website_hr_recruitment.tour', function(require) {
     'use strict';
 
     var tour = require("web_tour.tour");
+    function applyForAJob(jobName, applicantInformation) {
```

Comment:
It makes sense to have put this in a dedicated commit but then add more reference in the commit message (mentioning the commit that actually fixed the bug for which you add a test otherwise, we wonder why you add new test in stable without fixing anything).